Specs/DateTime/Hindi/TimeExtractor.json

```diff
@@ -324,47 +324,47 @@
     ]
   },
   {
-    "Input": "मैं दिन में 7 बजे वापस जाउंगा",
+    "Input": "मैं अपराहन में 7 बजे वापस जाउंगा",
```

Comment:
What's the reason for these multiple sentence changes and previously supported types?
Whenever a PR includes such changes, can you add a comment in the PR description?

Reply:
sorry, forgot to add comment. 
These cases were changed because the original translation was not accurate. In this specific case, it meant "I'll be back at 7 in the day" instead of "I'll be back at 7 pm"